active-directory-wpf-msgraph-v2/MainWindow.xaml.cs

```diff
@@ -56,7 +56,7 @@ private async void CallGraphButton_Click(object sender, RoutedEventArgs e)
                     authResult = await app.AcquireTokenInteractive(scopes)
                         .WithAccount(firstAccount)
                         .WithParentActivityOrWindow(new WindowInteropHelper(this).Handle) // optional, used to center the browser on the window
-                        .WithPrompt(Prompt.SelectAccount)
+                        .WithPrompt(Prompt.NoPrompt)
```

Comment:
Actually, without the select account, if I sign-out, and Select the **Call Microsoft Graph** button, I'm not given the choice of choosing the account, I'm systematically and automatically signed-in with the account.

So I'm not sure this is the experience we want?